index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="zxx" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/stylesheet.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@500&display=swap" rel="stylesheet">
+    <title>Adopt A Dog</title>
+  </head>
+
+  <body>
+
+
+    <header>
+      <div class="header-figures">
+
+        <figure class="company-logo">
+          <img src="images/adoptadog-logo.svg" alt="Company Logo">
+          <figcaption class="company">Seattle’s premiere dog adoption agency</figcaption>
+        </figure>
+
+        <figure class="cart-icon">
+          <img src="images/cart-icon.svg" alt="Cart Icon">
+          <a class="total" href="checkout.html" class="checkout">Checkout</a>
```

Comment:
Your cart should display a total. So as you click adopt, the cart total is reflected (using jQuery) here